### PR TITLE
Buhtiyarov p hw eng 1

### PR DESCRIPTION
**Pull request by puhlikov**
homework has been done

### CODE REVIEW

File: homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md [hunk 1]
```diff
@@ -2,48 +2,48 @@
...
-22. Maila (not to watch) TV every morning.
-23. Listen! Blanca (sing) in the bathroom.
-24. Bea sometimes (read) comics.
+1. Your mum (work) today?                            --- Is your mum working today?
+2. Why this boys (to talk) now?                      --- Why are this boys talking now?
+3. Where he (to live)?                               --- Where does he live?
+4. Their mother (cook) very well?                    --- Does their mother cook very well?
+5. Lili is not rich, she (not to drive) a Mercedes.  --- Lili is not rich, she doesn't drive a Mercedes.
+6. Let´s go out. It (not rain) now.                  --- Let's go out. It isn't raining now.
+7. I have to go now. It (get) dark                   --- I have to go now. It is geting dark.
```
Comment (goldenas18) [inline homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md:11]: looks like present simple. its a fact that it goes dark every day

File: homework/english/BuhtiyarovP_hw_eng_1/lesson1/vocabulary.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+5. Please, delete the comment.
+6. Reading develops the mind.
+7. I can disable the programm from here.
+8. You must to disconnect from server.
+9. Programm were displaed in the window computer.
+10. Television enables us to know what is happening today.
+11. I don't like to execute the commands.
+12. He may implement some efficient systems.
+13. I tried to install a new browser.
+14. We can't to plug in this monitor.
```
Comment (goldenas18) [inline homework/english/BuhtiyarovP_hw_eng_1/lesson1/vocabulary.md:19]: without "to". but in general seems ok to me

**Review by goldenas18 — DISMISSED** [review]

File: homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md (hunk 1 above)
Comment (ShelestV) [reply homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md:11]: No, this is not Present Simple
"Сейчас темнеет", this is no fact

File: homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+21. Mike (not wear) a tie today.                     --- Mike isn't wearing a tie today.
+22. Maila (not to watch) TV every morning.           --- Maila doesn't watch TV every morning.
+23. Listen! Blanca (sing) in the bathroom.           --- Listen. Blanca is singing in the bathroom.
+24. Bea sometimes (read) comics.                     --- Bea sometimes reads comics.
+
+### Найдите ошибки и исправьте их.
+1. We not going to school today.                     --- We aren't going to school today.
+2. What you doing after school?                      --- What are you doing after school?
+3. At the moment Peter is work in Russia.            --- At the moment Peter is working in Russia.
+4. Does he got a new car?                            --- Does he got a new car?
```
Comment (ShelestV) [inline homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md:34]: Verb "got" uses in incorrect form (II form)
You should "Does he get a new car?"

File: homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+5. He never wear a hat.                              --- He never don't wear a hat.
```
Comment (ShelestV) [inline homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md:35]: This is no Russian!
Use or "never", either "don't"
You should write "He never wears a hat."

**Review by ShelestV — APPROVED** [review]
Look tasks again!

File: homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md (hunk 1 above)
Comment (goldenas18) [reply homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md:11]: agree, checked that yesterday

**Review by breackinumber — APPROVED** [review]
Resolved

**Review by Littledrop777 — APPROVED** [review]

**Review by goldenas18 — APPROVED** [review]

**Pull request merged**